masterlist.yaml

```
@@ -26572,14 +26572,36 @@ plugins:
       - crc: 0xE4F0149C
         util: 'TES4Edit v4.0.4c'
 
+  - name: 'WAC - No Custom Races( CRR| GLAD)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/48276/'
+        name: 'Beards in Tamriel for WAC (No More Custom Races)'
   - name: 'WAC - No Custom Races.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/48276/' ]
     after:
       - 'WAC - TCOS.esp'
       - 'WAC - TCOS Shops.esp'
-    # doNotClean
     req: [ *Blockhead ]
     tag: [ NPC.Race ]
+    dirty:
+      - <<: *quickClean
+        crc: 0x2FB40EA4
+        util: '[TES4Edit v4.0.4c](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 13
```

The ITMs appear to be for reversing NPC race changes made in `WAC - TCOS.esp`, so we probably shouldn't display cleaning data for it. See #81 for where the comment (originally a message) was added, & I've also manually verified this.

```suggestion
    # Do not clean - ITMs are to reverse NPC race changes of `WAC - TCOS.esp`
```